lib/plugins/package/lib/packageService.js

```diff
@@ -69,14 +69,16 @@ module.exports = {
   async packageService() {
     legacy.log('Packaging service...');
     progress.get('main').notice('Packaging');
+    log.info('\nPackaging');
```

Comment:
It's safer to do:

```javascript
log.info();
log.info('Packaging');
```

Thing is, that if e.g. later we will decide to prefix verbose logs (e.g. with timestamp), then for `log.info('\nPackaging')` we will have timestamp and text after the new line

Reply:
good call :+1: